eth/tracers/internal/tracetest/firehose/firehose_test.go

```diff
@@ -38,25 +38,28 @@ func TestFirehosePrestate(t *testing.T) {
 		"./testdata/TestFirehosePrestate/extra_account_creations",
 	}
 
-	for _, folder := range testFolders {
-		name := filepath.Base(folder)
+	// TODO: have goroutine config on AND off
+	for _, concurrent := range []bool{true, false} {
+		for _, folder := range testFolders {
+			name := filepath.Base(folder)
 
-		for _, model := range tracingModels {
-			t.Run(string(model)+"/"+name, func(t *testing.T) {
-				tracer, tracingHooks, onClose := newFirehoseTestTracer(t, model)
-				defer onClose()
+			for _, model := range tracingModels {
+				t.Run(string(model)+"/"+name, func(t *testing.T) {
```

Comment:
The sub test name should be unique, you need to add the concurrency component somehow in the name.